lbry/blockchain/sync/synchronizer.py

```diff
@@ -43,9 +43,19 @@ def __init__(self, chain: Lbrycrd, db: Database):
         self.advance_loop_task: Optional[asyncio.Task] = None
         self.advance_loop_event = asyncio.Event()
 
+    async def wait_for_chain_ready(self):
```

Comment:
if `lbrycrd` was started without zmq enable would this just hang forever?

Reply:
yes. from the original report: `ZMQ readiness detection (currently just quits if lbrycrd isn't ready for ZMQ requests)`
so it wont give up until its ready, unless i misunderstood it

currently it waits in silence but thats because logging is broken. there is code on the exception handler to log what happened while it waits

Reply:
The method name is misleading then because it makes it seem like this waits for lbrycrd to be ready (in general) but in fact it's only waiting for zmq to be available... which may not be enabled, so lbrycrd would be ready (but not ZMQ).

Reply:
changed to instruct which conf key fixes it and exit in this case: `Lbrycrd is misconfigured. Please double check if zmqpubhashblock is properly set on lbrycrd.conf`